std_msgs/Header header
bool v_act_enabled
bool h_act_enabled
bool v_act_homed
bool h_act_homed
std_msgs/String debug